01-hello-git/README.md

```diff
@@ -66,194 +67,191 @@ or receiving changes from other repositories.
 
 ## Exercises
 
-### Creating a Fork
 
-A _fork_ is a personal copy of a repository hosted by a provider such as
-[GitHub](https://github.com). Create a fork and clone it.
+### Installing Git
+Before you start using Git, you have to make it available on your computer.
+click [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for 
+instulling on Linux, Mac and Windows.
+
+### Tell Git who you are
+First, you need to tell Git who you are:
 
 ```
-$ git clone git@github.com:jawnsy/training.git
-$ cd training
-$ ls
+$ git config --global user.email "you@example.com"
+$ git config --global user.name "Your Name"
+```
+
+### Give GitHub your public keys
+
+1- Paste the text Below, Substituting in your GitHub email address:
+
+```
+$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+```
+
+This creates a new ssh key, using the provided email as a label.
+
+
 ```
+Generating public/private rsa key pair.
 
-_Tip_: We recommend using an [_SSH Key_](https://help.github.com/articles/connecting-to-github-with-ssh/)
-to authenticate.
+```
+2- When you're prompted to "Enter a file in which to save the key," press Enter. 
+This accepts the default file location.
 
-_Stuck?_ See the [GitHub documentation](https://help.github.com/articles/fork-a-repo/)
-for help.
 
-### Configure Git
+```
+Enter a file in which to save the key (/c/Users/you/.ssh/id_rsa):[Press enter]
 
-Configure Git with your full name and email address:
+```
+3- At the prompt, type a secure passphrase. Your passphrase is not visible in Git Bash window.
 
 ```
-$ git config --global user.name "Jonathan Yu"
-$ git config --global user.email "Jonathan.Yu@smartwires.com"
-$ git config user.name
-Jonathan Yu
-$ git config user.email
-Jonathan.Yu@smartwires.com
+Enter passphrase (empty for no passphrase): [Type a passphrase]
+Enter same passphrase again: [Type passphrase again]
 ```
 
-This configures the default name and email address to use in commit messages.
-The `--global` flag causes the setting to apply to all repositories on your
-system, but per-repository configuration can override these. For more detail,
-see [Customizing Git](https://git-scm.com/book/tr/v2/Customizing-Git-Git-Configuration).
 
-### Making Changes
```

Comment:
The original exercises were deleted, was that intentional?

Reply:
I added this part to the edited README file.